docs/developer/checkout/performance.mdx

```diff
@@ -28,4 +28,64 @@ You can leverage strategy called "optimistic updates" for that.
 - [React Query](https://tanstack.com/query/v4/docs/framework/react/guides/optimistic-updates)
 - [urql](https://commerce.nearform.com/open-source/urql/docs/graphcache/cache-updates/#optimistic-updates)
 
-In the nutshell, frontend can calculate new price locally and update the UI immediately, while the real response will apply with a delay in the background.
\ No newline at end of file
+In the nutshell, frontend can calculate new price locally and update the UI immediately, while the real response will apply with a delay in the background.
+
+### Lazy vs eager recalculations
+
+Some mutations require price recalculations - such as changing lines quantity or adding a voucher. However, not every mutation behaves the same way.
+
+With performance considerations, there are two strategies how mutations recalculate prices:
+
+- Eager: Mutation will call `*_CALCULATE_TAXES` event during the mutation execution. That means mutation will run slower and the returned value will have latest updates prices values.
```

Comment:
**Suggestion:** maybe we can add links to `CHECKOT_CALCULATE_TAXES` and `ORDER_CALCULATE_TAXES` - it will be easier to search / discover and there are only two such events